undebt/pattern/interface.py

```diff
@@ -78,10 +84,23 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
+    if module_like(path):
+        path = module_name_to_path(path)
```

Comment:
If only I had looked up to check my email!! I'll switch over to `importlib`, which presumably does pretty much what I just implemented :sigh: